crates/ruff_python_formatter/tests/fixtures.rs

```diff
@@ -177,12 +177,9 @@ fn format() {
     let test_file = |input_path: &Path| {
         let content = fs::read_to_string(input_path).unwrap();
 
-        let options = PyFormatOptions::from_extension(input_path);
-        let formatted_code = format_file(&content, &options, input_path);
```

Comment:
Ha, that took me a minute to figure out. We always formatted the code once with the default options even if there's an options file. We don't want that, so I moved it into the `else` branch`